### DISCUSSION TITLE
SetFontScheme nowhere to be found?

### DISCUSSION THREAD

**Discussion by StanislawFlexpedia** [ClosedXML/ClosedXML#2005] [Q&A] [answered]
I dont know what im doing wrong, but im not able to set font scheme.
Example from [documentation](https://closedxml.readthedocs.io/en/latest/features/cell-format.html#font-scheme) ([documentation2](https://closedxml.readthedocs.io/en/latest/api/index.html#enumerator-ClosedXML.Excel.XLFontScheme.Minor)) im trying to replicate:
```
cell.GetRichText()
    .AddText("Major scheme")
        .SetFontName("Arial")
        .SetFontScheme(XLFontScheme.Major)
```

**Comment by jahav** [comment, marked as answer]
That is not yet released, so it is only in develop branch. ReadTheDocs is set to build doc from develop branch atm.

**Reply by StanislawFlexpedia** [reply]
is it possible to get it through nuget by any chance?

**Reply by jahav** [reply]
Only if you make your own feed. https://github.com/ClosedXML/ClosedXML/wiki/Development-Builds

**Reply by StanislawFlexpedia** [reply]
sad, ill wait for new release then. Thank you
